### PR TITLE
item for consistent messaging / esl consideration

### PR DESCRIPTION
**Pull request by scottaohara**
reading through the list, this didn't jump right out at me.  but if this is already meant to be implied by another point, then please do ignore :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,4 @@ Aims to be the **biggest** checklist of inclusive design considerations ever. In
 - [ ] Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab
 - [ ] [Make content easier to find and improve search results with structured data](https://developers.google.com/search/docs/guides/prototype)
 - [ ] Use textual labels to make voice activation cues obvious
+- [ ] Esnure important calls to action are consistent and easily translatable to other languages
```
Comment (adamsilver) [inline README.md]: Typo: “Esnure” -> ”Ensure”

Comment (scottaohara) [reply README.md]: typing in haste is not my friend.  thank you!

**Comment by Heydon** [comment]
Not sure about "important". Surely all actions should be translatable?

**Comment by scottaohara** [comment]
good point, @Heydon.  revised to be more general.

**Pull request merged**